tests/integration/cli_test_util.go

```diff
@@ -0,0 +1,133 @@
+package integration
```

Comment:
These aren't real integration tests--they mock everything. Overall, I'm not a fan of mocking unless it's *absolutely necessary*. It just becomes too much of an opportunity to contrive conditions that aren't real and becomes a huge time vampire because of the complexity involved and having to make design decisions just to make things easier to mock. I suggest using the real CLI for true integration tests.

Reply:
thanks @nathanagood this was a good conversation starter. 
We talked a bit offline -- I think there's a path forward here were we minimize mocking as much as possible, while still being able to run tests locally (without a deployed DCE API environment). Though we also want to start running full E2e tests with the dce-cli